# RNA_stuff  
In this README you can find diverse information about typical RNA sites, meetings, tools, databases, etc.

Please, feel free to improve and add informatition.

## RNA typical web-sites
* [The RNA society site](https://www.rnasociety.org/), and their [twitter](https://twitter.com/rnasociety)
* [Sweden RNA site](http://www.rnass.se/) (look the researchers section if there is anyone known)
* [The RNA institute](https://www.rna.albany.edu)
* [The G4 Society](https://www.g4-society.org/)

## RNA anual meetings:
* [2020 CANADA](https://www.rnasociety.org/conferences/rna-2020/meeting-information/), from RNA society
* [2019 SWEDEN](https://swedishrnasocietymeeting.se/), rna society in sweden (made in Sep-2019, maybe in 2020 could be the next one)
* [Rustbelt](https://rustbeltrna.org/)
* [2020 USA](https://www.rna.albany.edu/7th-annual-rna-symposium/#top), The RNA institute meeting
* [2020 SWEDEN](https://www.irt2020.se/), IRT NNN 

## RNA structural competition
* [RNA Puzzles](http://www.rnapuzzles.org/), a CASP-like.
  * [Recent paper](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkz1108/5651330) (2019-12-04)
  
## Common databases:
* **Sequences**
  * [Rfam](https://rfam.xfam.org/), (RNA families)
  * [RNACentral](https://rnacentral.org/), (non-coding RNAs from 40 curated databases)
* **Structures**
  * [PDB](https://www.rcsb.org/) and [NDB](http://ndbserver.rutgers.edu/)
  * [RNArchitecture](http://iimcb.genesilico.pl/RNArchitecture/)
  * [SCOR](http://scor.berkeley.edu/scor.html) (down)
  * [URS](http://server3.lpm.org.ru/urs/)
  * [BGSU RNA site](http://rna.bgsu.edu/rna3dhub/)
  * [Cosmos database](http://rnacossmos.com/)
  * [Metal PDB](http://metalweb.cerm.unifi.it/)
  * [RNA topology](http://www.biomath.nyu.edu/?q=rag/home) using graph theory
  * [RNA virus](http://viperdb.scripps.edu/index.php) ([new v3](http://dante.scripps.edu/))
  * [5S rRNA Database](http://combio.pl/rrna/)
* **Residues**
  * [The RNA modification database](https://mods.rna.albany.edu/home)
  * [RNA basepair database](http://hdrnas.saha.ac.in/rnabpdb/)
  * [RNABP COGEST](http://bioinf.iiit.ac.in/RNABPCOGEST/) (has another links basepair databases inside)
* **Others**
 * [NCRNA Databases](https://ncrnadatabases.org/)
 * [3D RNA rotamers](http://kinemage.biochem.duke.edu/research/rnarotamer.php) (Richardsons Lab)
 * [Chemical mapping data](https://rmdb.stanford.edu/)

## Groups:
* [Bujnicki](http://genesilico.pl/) (one of the best, has softwares, databases, etc)
* [Znosko](https://www.slu.edu/~znoskob/)


## Tools:
* [X3DNA](http://x3dna.org/) ([Xiang-Jun Lu](https://systemsbiology.columbia.edu/people/xiang-jun-lu), [DSSR forum](http://forum.x3dna.org/rna-structures/), [web 3XDNA](http://web.x3dna.org/))
* [rna-tools](https://rna-tools.readthedocs.io/en/latest/index.html) ([Magnus](https://mmagnus.github.io/) from Bujnicki group)
* [R3D-BLAST2](http://140.114.85.168/R3D-BLAST2/) (search substructures using structural alphabet)
* [RNAflex](http://sparks-lab.org/server/RNAflex/) (using SVM)
* [RBP Motif Search](https://github.com/nberkow/RBP_Motif_Search)

## Papers:
* RNA 3D prediction
  * Homology
    * [RNA 3D structure prediction guided by independent folding of homologous sequences.](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-3120-y)
* RNA 3D contacts
  * [ClaRNA: a classifier of contacts in RNA 3D structures based on a comparative analysis of various classification schemes.](https://www.ncbi.nlm.nih.gov/pubmed/25159614?dopt=Abstract)
